dashboard/templates/cards/sleep_recent.html

```diff
@@ -0,0 +1,54 @@
+{% extends 'cards/base.html' %}
+{% load duration i18n %}
+
+{% block header %}
+    <a href="{% url "core:sleep-list" %}">
+        {% trans "Today's Sleep" %}
```

Comment:
Lets change this is to "Recent Sleep" as well.

Reply:
Ok, Does that mean we need to go through each *.po file to update the localized String?

Reply:
Yeah but I can handle that in a separate commit.

If you want to you can run `gulp makemessages -a` and then `gulp compilemessages` but sometimes the message stuff needs some fiddling (I think it _shouldn't_ in this case).

Reply:
ran the messages command. The below had a newline in ES (line 791 in es/django.po), so I added it in EN as well since it was throwing an error.

`"Add <samp>%(origin)s</samp> to the <code>CSRF_TRUSTED_ORIGINS</code> "
"environment variable. If multiple origins are required separate with commas.\n"`